pkg/transforms/genericResourceConfig.go

```diff
@@ -7,6 +7,8 @@ type ExtractProperty struct {
 	DataType DataType // `json:"dataType,omitempty"`
 	// An internal property to denote this property should be set on the node's metadata instead.
 	metadataOnly bool
+	// A property to denote that we should gather status conditions on this resource
+	StatusConditions bool
```

Comment:
I prefer this flag on the `ResourceConfig` object.

The `name` and `JSONPath` are always constant. It's better to move the flag a level above.
```
{Name: "conditions", JSONPath: `{.status.conditions}`
```